lib/igniter/libs/phoenix.ex

```diff
@@ -199,6 +199,8 @@ defmodule Igniter.Libs.Phoenix do
   * `:router` - The router module to append to. Will be looked up if not provided.
   * `:arg2` - The second argument to the scope macro. Must be a value (typically a module).
   * `:with_pipelines` - A list of pipelines that the pipeline must be using to be considered a match.
+  # TODO: improve docs
+  * `:placement` - :before_first_scope | :bottom_of_module - defaults to :before_first_scope (current behavior)
```

Comment:
I think maybe it could be as simple as `:first` or `:last`, and we can evolve from there. We can't guarantee much past that anyway since the router could have all kinds of stuff in it. But we can at least guarantee that it goes after all other routes & scopes or before all other routes and scopes.